gp-includes/formats/format_pomo.php

```diff
@@ -45,6 +51,36 @@ public function read_originals_from_file( $file_name ) {
 		return $this->read_translations_from_file( $file_name );
 	}
 
+	/**
+	 * Create a string that represents the value for the "Language:" header for a po file.
+	 *
+	 * @since 2.1.0
+	 *
+	 * @param GP_Locale $locale The locale object.
+	 *
+	 * @return string|false
+	 */
+	private function get_language_code( $locale ) {
+		$ret = '';
+
+		if ( $locale->lang_code_iso_639_1 ) {
+			$ret = $locale->lang_code_iso_639_1;
+		} elseif ( $locale->lang_code_iso_639_2 ) {
+			$ret = $locale->lang_code_iso_639_2;
+		} elseif ( $locale->lang_code_iso_639_3 ) {
+			$ret = $locale->lang_code_iso_639_3;
+		}
+
+		if ( '' === $ret ) {
+			return false;
+		}
+
+		if ( null !== $locale->country_code && strcasecmp( $ret, $locale->country_code ) === 0 ) {
```

Comment:
Yes, 'de_DE' == 'de' so the code skips adding it if they're the same.  When they're not the same (like de_CH) it gets added.